src/components/hero/hooks.ts

```diff
@@ -0,0 +1,34 @@
+import { MuiTabProps } from 'types';
+import { StringParam, useQueryParam, withDefault } from 'use-query-params';
+import { TabOptions } from './types';
+
+export const TABS: MuiTabProps<TabOptions>[] = [
+    {
+        label: 'home.hero.tab.companyOverview',
+        value: 'overview',
+    },
+    {
+        label: 'home.hero.tab.companyDetails',
+        value: 'details',
+    },
+    {
+        label: 'home.hero.tab.demo',
+        value: 'demo',
+    },
+];
+
+export function useHeroTabs() {
+    const [activeTab, setActiveTab] = useQueryParam(
+        'activeTab',
+        withDefault(StringParam, TABS[0].value)
+    );
+
+    return {
+        activeTab,
+        setActiveTab,
+        openDetails: () => {
+            setActiveTab('details');
+        },
```

Comment:
Decided to make a single helper because it was already made elsewhere. Feels kinda weird give we can just pass the tab we want open but felt it as easiest with the conversion.